webapi/_test_data/all_data/admin-data.ttl

```diff
@@ -470,3 +470,41 @@ Die Internetpublikation macht das digitalisierte Korpus dieser Frühdrucke  durc
   knora-base:isInProject <http://rdfh.ch/projects/DczxPs-sR6aZN91qV92ZmQ> ;
   knora-base:isInSystemAdminGroup "false"^^xsd:boolean ;
   knora-base:isInProjectAdminGroup <http://rdfh.ch/projects/DczxPs-sR6aZN91qV92ZmQ> .
+
+
+###################################
+# WEBERN                          #
+###################################
+
+<http://rdfh.ch/projects/08AE> a knora-base:knoraProject ;
+  knora-base:projectShortname "webern"^^xsd:string ;
+  knora-base:projectLongname "Anton Webern Gesamtausgabe"^^xsd:string ;
```

Comment:
Can you please assign a `projectShortcode`

Reply:
Done